recipes-devtools/dediprog-flasher/dediprog-flasher/0002-Makefile-use-the-STRIP-env-var.patch

```diff
@@ -0,0 +1,26 @@
+From b726eb61f8ab97747de2db3903d8879ca2472592 Mon Sep 17 00:00:00 2001
+From: Tymoteusz Burak <tymoteusz.burak@3mdeb.com>
+Date: Mon, 8 Jul 2024 10:38:42 +0000
+Subject: [PATCH] Makefile: use the $STRIP env variable
+
+ Signed-off-by: Tymoteusz Burak <tymoteusz.burak@3mdeb.com>
+---
+ Makefile | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/Makefile b/Makefile
+index 139e5c0..89bdc16 100644
+--- a/Makefile
++++ b/Makefile
+@@ -58,7 +58,7 @@ install: $(PROGRAM)
+ 	[ $(shell id -u) -eq 0 ] || (echo "Error: install needs root privileges" && false)
+ 	install -v -o 0 -g 0 -m 755 -d $(DESTDIR)$(PREFIX)/bin $(DESTDIR)$(PREFIX)/share/DediProg
+ 	echo -n "install: " && install -v -o 0 -g 0 -m 0755 $(PROGRAM) $(DESTDIR)$(PREFIX)/bin/$(PROGRAM)
+-	strip $(DESTDIR)$(PREFIX)/bin/$(PROGRAM)
++	$(STRIP) $(DESTDIR)$(PREFIX)/bin/$(PROGRAM)
```

Comment:
@tym2k1 I think you should also set the default value of STRIP to strip so you do not break existing functionality